src/features/Layout/Navigation/DesktopNav.tsx

```diff
@@ -64,8 +64,8 @@ const DesktopNav = ({ appRoutes, current, openTray }: IDesktopNav) => {
       </Box>
       <NavLink link={settings} current={current === settings.to} />
       <SBox variant="columnCenter" width="100%" px="3px" py="12px" onClick={openTray}>
-        <Icon type="nav-menu" height="2.5vh" color="WHITE" />
-        <Text variant="navItem" fontSize={{ _: '10px', sm: '1.1vh' }} color="WHITE">
+        <Icon type="nav-menu" height={{ sm: '2.4vh', xxl: '24px' }} color={COLORS.WHITE} />
```

Comment:
Generates flood of errors in the console.
See transient props or `shouldForwardProp` https://styled-components.com/docs/api#transient-props

<img width="368" alt="Screenshot 2021-01-25 at 13 55 47" src="https://user-images.githubusercontent.com/2429708/105708649-08b58d80-5f15-11eb-98fa-cf733d4ca4ea.png">